geonode/templates/500.html

```diff
@@ -196,7 +196,7 @@ <h3>{% trans "There was a problem loading this page" %}</h3>
           {% block contactus %}
           {% endblock contactus %}
           <ul class="list-unstyled">
-            <li>{% trans "Powered by" %} <a href="http://geonode.org">{% trans "GeoNode" %}</a> <em>{% trans "version" %} {{ VERSION }}</em></li>
+            <li>{% trans "Powered by" %} <a href="http://geonode.org">{% trans "GeoNode" %}</a>{% if user.is_staff %} <em>{% trans "version" %} {{ VERSION }}</em>{% endif %}</li>
```

Comment:
In a perfect world we would have this part of the footer once and the responseCode.htmls would import it?